nais/dev/dev-gcp-mock.env

```diff
@@ -0,0 +1,5 @@
+REACT_APP_ENVIRONMENT="dev-gcp"
```

Comment:
Burde denne hete `dev-gcp-mock`? Foreløpig bruker vi kun denne for å skille på tracing-data i Sentry.

Reply:
Usikker. Er det behov for å skille?